src/Agent.Plugins/TfsVCSourceProvider.cs

```diff
@@ -253,7 +253,8 @@ public sealed class TfsVCSourceProvider : ISourceProvider
                 tfWorkspaces = await tf.WorkspacesAsync(matchWorkspaceNameOnAnyComputer: true);
                 foreach (ITfsVCWorkspace tfWorkspace in tfWorkspaces ?? new ITfsVCWorkspace[0])
                 {
-                    await tf.WorkspaceDeleteAsync(tfWorkspace);
+
+                    await tf.TryWorkspaceDeleteAsync(tfWorkspace);
```

Comment:
Could you please fix the code formatting?
```suggestion
                    await tf.TryWorkspaceDeleteAsync(tfWorkspace);
```